### PR TITLE
Pass binary abis to sign function

### PR DESCRIPTION
**Pull request by randytorres**

### CODE REVIEW

**Comment by randytorres** [comment]
The binary abi is being passed to the sign function, the function is not doing anything with the abi right now.

File: src/eosjs2-jsonrpc.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export interface Abi {
 /** Return value of `/v1/chain/get_abi` */
 export interface GetAbiResult {
   account_name: string;
-  abi: Abi;
+  abi: Abi | Uint8Array;
```
Comment (tbfleming) [inline src/eosjs2-jsonrpc.ts]: `/v1/chain/get_abi` doesn't return a `Uint8Array`. Define a new interface in `eosjs2-api.ts` instead.

**Comment by chris-allnutt** [comment]
@randytorres looks like this needs a conflict resolution

**Comment by brandonfancher** [comment]
@randytorres This looks good. We'll need to update a few consumers of this API down the line, but that can be done when we cut a new npm release, and the changes should be fairly straightforward. So I approve going ahead and merging after conflict resolution.

**Pull request merged**